webapp/src/pages/LedgersOverview.jsx

```diff
@@ -109,12 +109,16 @@ export default function LedgersOverview() {
             linesLoading={ledgersOverviewLoading || ledgerLinesLoading}
           />
           <LayoutContainer gutters>
-            <ForwardBackPagination
-              page={page}
-              pageCount={ledgerLines.pageCount}
-              onPageChange={(pg) => setListQueryParams({ page: pg })}
-              scrollTop={140}
-            />
+            {!isEmpty(activeLines) ? (
+              <ForwardBackPagination
+                page={page}
+                pageCount={ledgerLines.pageCount}
+                onPageChange={(pg) => setListQueryParams({ page: pg })}
+                scrollTop={140}
+              />
+            ) : (
+              <p className="text-center">{t("payments:no_transaction_history")}</p>
```

Comment:
This isn't true, so is confusing- they can be looking at an empty ledger but have spent money elsewhere. Either rephrase or remove.